tft.py

```diff
@@ -219,6 +219,12 @@ def queue() -> None:  # pylint: disable=too-many-branches
         if not PLAY_NEXT_GAME:
             evaluate_next_game_logic()
 
+        if LCU_INTEGRATION.session_expired():

```

Comment:
Nice, much cleaner -- I didn't think we could pull that from the LCU.

Reply:
I'm using this great tool called LCUExplorer which lets you play around with the LCU through swagger, it's definitely worth a look if you ever have some time :) There's some really neat endpoints